.github/workflows/pull_request_verification.yml

```diff
@@ -20,6 +20,23 @@ jobs:
       - name: Check correct commit message
         run: ./.github/scripts/commit/checkCommitMessage.sh
 
+  count-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR commits
+        run: echo "PR_FETCH_DEPTH=$(( ${{ github.event.pull_request.commits }} ))" >> "${GITHUB_ENV}"
```

Comment:
Is thins needed as a separate step? Are we able to merge it with `Count Commits` step?

Reply:
I will try it

Reply:
Done 😄